Documentation/data-type-and-flow-control.md

```diff
@@ -0,0 +1,121 @@
+---
+title: Data type and flow control
+...
+
+[Rank counting with other/tensor types](Documentation/rank-counting-with-other-tensor.md)
+
+# GStreamer data types (pad capabilities)
+
+All NNStreamer's GStreamer data types as pad capabilities (```other/tensor*```) have the following common rules
+
+1. In each buffer, there is only ONE frame. That is for each buffer, with the type of ```other/tensor```, there is only one instance of tensor for each buffer at any time. There cannot be multiple tensors in each buffer.
+2. The data types do not hold data semantics. Filters should NOT try to determine data semantics (e.g., is it a video?) dynamically based solely on the dimensions, framerates, or element types of the data types. However, if a filter has additional information available including property values from pipeline developers or users, a filter may determine data semantics. For example, ```tensor_decoder``` transforms ```other/tensor``` stream into ```video/x-raw``` or ```text/x-raw``` depending on the property values.
+
+## other/tensor
+
+The GST pad capability has the following structure:

```

Comment:
The GStreamer pad capability has ...
compated to line 60 (The GStreamer pad capability of ```other/tensors``` is as follows:)

Reply:
Nice! updated.